### PR TITLE
Update documents that are related to schema loader

### PR DESCRIPTION
**Pull request by thongdk8**
This PR updates documents that mention schema tool which is changed to a new schema loader. PTAL. Thank you!

### CODE REVIEW

File: docs/getting-started-with-scalardb.md [hunk 1]
```diff
@@ -48,29 +48,12 @@ You can create a JSON file `emoney-storage.json` with the JSON below.
...
-$ java -jar scalar-schema-standalone-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f emoney-storage.json
-```
-  - `<REGION>` should be a string to specify an AWS region like `ap-northeast-1`.
-
-For JDBC databases
-```
-$ java -jar scalar-schema-standalone-<version>.jar --jdbc -j <JDBC_URL> -u <USERNAME> -p <PASSWORD> -f emoney-storage.json
+$ java -jar scalardb-schema-loader-<version>.jar --config <path_to_database.properties_file> -f emoney-storage.json
 ```
+- The **database.properties** file contains the configurations for Scalar DB which was mentioned in **Getting Started with Scalar DB on X** documents.
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:56]: I think `Getting Started with Scalar DB on X` should be linked.

**Review by brfrn169 — COMMENTED** [review]
Thank you! Left several minor suggestions. Please take a look!

File: docs/getting-started-with-scalardb.md [hunk 2]
```diff
@@ -306,7 +267,7 @@ These are just simple examples of how Scalar DB is used. For more information, p
     * [scalardb-server](https://javadoc.io/doc/com.scalar-labs/scalardb-server/latest/index.html) - Scalar DB Server that is the gRPC interfarce of Scalar DB
 * [Requirements in the underlining databases](requirements.md)
 * [Database schema in Scalar DB](schema.md)
-* [Schema tool](../tools/scalar-schema/README.md)
+* [Schema loader](../schema-loader/README.md)
```
Comment (brfrn169) [inline docs/getting-started-with-scalardb.md:270]:
```suggestion
* [Schema loader](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/README.md)
```

File: docs/getting-started-with-scalardb.md (hunk 1 above)
Comment (thongdk8) [reply docs/getting-started-with-scalardb.md]: I updated it in [commit](https://github.com/scalar-labs/scalardb/pull/352/commits/4d0a25733cc01baf98adfd8c8a1b41b3815fa206)

File: docs/getting-started-with-scalardb.md [hunk 3]
```diff
@@ -48,29 +48,12 @@ You can create a JSON file `emoney-storage.json` with the JSON below.
...
-$ java -jar scalar-schema-standalone-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f emoney-storage.json
-```
-  - `<REGION>` should be a string to specify an AWS region like `ap-northeast-1`.
-
-For JDBC databases
-```
-$ java -jar scalar-schema-standalone-<version>.jar --jdbc -j <JDBC_URL> -u <USERNAME> -p <PASSWORD> -f emoney-storage.json
+$ java -jar scalardb-schema-loader-<version>.jar --config <path_to_database.properties_file> -f emoney-storage.json
 ```
+- The **database.properties** file contains the configurations for Scalar DB which was mentioned in **Getting Started with Scalar DB on X** documents, for example [Getting Started with Scalar DB on Cassandra](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-cassandra.md).
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:56]:
```suggestion
```
I think users come to this page after configuring the database.properties so I don't think this description is needed.

File: docs/getting-started-with-scalardb.md [hunk 4]
```diff
@@ -48,29 +48,12 @@ You can create a JSON file `emoney-storage.json` with the JSON below.
...
-For DynamoDB
-```
-$ java -jar scalar-schema-standalone-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f emoney-storage.json
-```
-  - `<REGION>` should be a string to specify an AWS region like `ap-northeast-1`.
-
-For JDBC databases
-```
-$ java -jar scalar-schema-standalone-<version>.jar --jdbc -j <JDBC_URL> -u <USERNAME> -p <PASSWORD> -f emoney-storage.json
+$ java -jar scalardb-schema-loader-<version>.jar --config <path_to_database.properties_file> -f emoney-storage.json
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:54]:
```suggestion
$ java -jar scalardb-schema-loader-<version>.jar --config /path/to/database.properties -f emoney-storage.json
```

File: docs/schema.md [hunk 5]
```diff
@@ -34,4 +34,4 @@ Thus, along with any required values by the application, additional values for t
 ## Schema creation
 
 It is a little hard for application developers to care for the schema mapping and metadata for transactions,
-so we offer a tool called [scalar-schema](../tools/scalar-schema/README.md) for creating schema without much knowledge about those.
+so we offer a tool called [Schema loader](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/README.md) for creating schema without much knowledge about those.
```
Comment (feeblefakie) [inline docs/schema.md:37]:
```suggestion
so we offer a tool called [Schema Loader](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/README.md) for creating schema without much knowledge about those.
```

File: docs/getting-started-with-scalardb.md [hunk 6]
```diff
@@ -306,7 +267,7 @@ These are just simple examples of how Scalar DB is used. For more information, p
     * [scalardb-server](https://javadoc.io/doc/com.scalar-labs/scalardb-server/latest/index.html) - Scalar DB Server that is the gRPC interfarce of Scalar DB
 * [Requirements in the underlining databases](requirements.md)
 * [Database schema in Scalar DB](schema.md)
-* [Schema tool](../tools/scalar-schema/README.md)
+* [Schema loader](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/README.md)
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:270]:
```suggestion
* [Schema Loader](https://github.com/scalar-labs/scalardb/tree/master/schema-loader/README.md)
```

File: docs/getting-started-with-scalardb.md [hunk 7]
```diff
@@ -166,33 +149,11 @@ You can create a JSON file `emoney-transaction.json` with the JSON bellow.
...
-$ java -jar scalar-schema-standalone-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f emoney-storage.json -D
-$ java -jar scalar-schema-standalone-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f emoney-transaction.json
-```
-
-For JDBC databases
-
 ```
-$ java -jar scalar-schema-standalone-<version>.jar --jdbc -j <JDBC_URL> -u <USERNAME> -p <PASSWORD> -f emoney-storage.json -D
-$ java -jar scalar-schema-standalone-<version>.jar --jdbc -j <JDBC_URL> -u <USERNAME> -p <PASSWORD> -f emoney-transaction.json
+$ java -jar scalardb-schema-loader-<version>.jar --config <path_to_database.properties_file> -f emoney-storage.json -D
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:153]:
```suggestion
$ java -jar scalardb-schema-loader-<version>.jar --config /path/to/database.properties -f emoney-storage.json -D
```

File: docs/getting-started-with-scalardb.md [hunk 8]
```diff
@@ -166,33 +149,11 @@ You can create a JSON file `emoney-transaction.json` with the JSON bellow.
...
+$ java -jar scalardb-schema-loader-<version>.jar --config <path_to_database.properties_file> --coordinator -f emoney-transaction.json
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:154]:
```suggestion
$ java -jar scalardb-schema-loader-<version>.jar --config /path/to/database.properties --coordinator -f emoney-transaction.json
```

File: docs/getting-started-with-scalardb.md [hunk 9]
```diff
@@ -48,29 +48,12 @@ You can create a JSON file `emoney-storage.json` with the JSON below.
 }
 ```
 
-To apply the schema, download the scalar schema standalone loader that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
+To apply the schema, download the Schema loader that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
```
Comment (feeblefakie) [inline docs/getting-started-with-scalardb.md:51]:
```suggestion
To apply the schema, download the Schema Loader that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
```

**Review by feeblefakie — APPROVED** [review]
LGMT! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**